packages/@ember/-internals/glimmer/tests/integration/modifiers/on-test.js

```diff
@@ -0,0 +1,324 @@
+import { EMBER_GLIMMER_ON_MODIFIER } from '@ember/canary-features';
+import { moduleFor, RenderingTestCase, runTask } from 'internal-test-helpers';
+import { isChrome, isFirefox } from '@ember/-internals/browser-environment';
+import { privatize as P } from '@ember/-internals/container';
+
+const isIE11 = !window.ActiveXObject && 'ActiveXObject' in window;
+
+if (EMBER_GLIMMER_ON_MODIFIER) {
+  moduleFor(
+    '{{on}} Modifier',
+    class extends RenderingTestCase {
+      beforeEach() {
+        // might error if getOnManagerInstance fails
+        this.startingCounters = this.getOnManagerInstance().counters;
+      }
+
+      getOnManagerInstance() {
+        // leveraging private APIs, this can be deleted if these APIs change
+        // but it has been useful to verify some internal details
+        let templateCompiler = this.owner.lookup(P`template-compiler:main`);
+
+        return templateCompiler.resolver.resolver.builtInModifiers.on.manager;
+      }
+
+      assertCounts(expected) {
+        let { counters } = this.getOnManagerInstance();
+
+        this.assert.deepEqual(
+          counters,
+          {
+            adds: expected.adds + this.startingCounters.adds,
+            removes: expected.removes + this.startingCounters.removes,
+          },
+          `counters have incremented by ${JSON.stringify(expected)}`
+        );
+      }
+
+      '@test SUPPORTS_EVENT_OPTIONS is correct (private API usage)'(assert) {
+        let manager = this.getOnManagerInstance();
+
+        let { SUPPORTS_EVENT_OPTIONS } = manager;
+
+        if (isChrome || isFirefox) {
+          assert.strictEqual(SUPPORTS_EVENT_OPTIONS, true, 'is true in chrome and firefox');
+        } else if (isIE11) {
+          assert.strictEqual(SUPPORTS_EVENT_OPTIONS, false, 'is true in chrome and firefox');
```

Comment:
```suggestion
          assert.strictEqual(SUPPORTS_EVENT_OPTIONS, false, 'is false in IE11');
```

Or was this intentional to keep the logout output the same?

Reply:
Nope, was a copy/paste mistake